PRS13001-14000/PRS13198Tewodros.xml

```diff
@@ -47,18 +47,19 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
                 <person sex="1">
                     <persName xml:lang="gez" xml:id="n1">ቴዎድሮስ</persName>
                     <persName xml:lang="gez" type="normalized" corresp="#n1">Tewodros</persName>
+                    <persName xml:lang="gez"><roleName type="title">kāhǝn</roleName></persName>
                     <faith type="EOTC"/>
                     <occupation type="literary"/>
+                    <occupation type="ecclesiastic">manakos</occupation>
```

Comment:
I think that giving the English "monk" might be appropriate here. I don't feel that the Geez term has technical meanings not covered by the English one.

Reply:
Yes.